test/org/zaproxy/zap/extension/pscanrules/TestInfoPrivateAddressDisclosureUnitTest.java

```diff
@@ -90,15 +90,27 @@ public void alertsIfPrivateIp() throws HttpMalformedHeaderException {
 	}
 
 	@Test
-	public void shouldIgnoreRequestedPrivateIpByDefault() throws HttpMalformedHeaderException {
+	public void shouldIgnoreRequestedPrivateIpByDefault() throws HttpMalformedHeaderException, URIException, NullPointerException {
 		// Given
-		String privateIp = "192.168.36.127";
-		String requestUri = "https://" + privateIp + ":8123/";
-		HttpMessage msg = createHttpMessage(requestUri, privateIp);
-		// When
-		rule.scanHttpResponseReceive(msg, -1, createSource(msg));
-		// Then
-		assertThat(alertsRaised, is(empty()));
+		// ip and aws-hostname which get concatenated with the ports as candidates
+		String[] ipHost = new String[] {"10.0.2.2", "iP-10-0-2-2"};
```

Comment:
I'm going to revert this change actually, per `passesIfInvalidAwsHostname()` is seems like it should be lower only.